advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java

```diff
@@ -152,53 +155,118 @@
 		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
 	}
 
+	/** FactoryMap collection.
+	 *
+	 */
+	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
+			Collections.unmodifiableMap(new HashMap<AttributeType, StringFactory>() {{
```

Comment:
I get error if I use one '{'. I have not given it much thought why. Probably due to the Collections we use encircling the HashMap. I will definetely try to find the main reason for that in due course.